src/autobind.cr

```diff
@@ -39,41 +66,43 @@ while arg = ARGV[i += 1]?
     when "true"      then remove_enum_suffix = true
     else                  remove_enum_suffix = value
     end
-  when "--help"
-    STDERR.puts <<-EOF
-    usage : c2cr [--help] [options] <header.h>
-
-    Some available options are:
-        -I<directory>   Adds directory to search path for include files
-        -D<name>        Adds an implicit #define
-
-    In addition, the CFLAGS environment variable will be used, so you may set it
-    up before compilation when search directories, defines, and other options
-    aren't fixed and can be dynamic.
-
-    The following options control how enum constants are cleaned up. By default
-    the value is false (no cleanup), whereas true will remove matching patterns,
-    while a fixed value will remove just that:
-        --remove-enum-prefix[=true,false,<value>]
-        --remove-enum-suffix[=true,false,<value>]
-    EOF
+  when .starts_with? "--lib-name"
+    abort "Only one library name can be specified", USAGE_EXIT_CODE unless lib_name == "LibC"
+    if arg.includes? '='
+      lib_name = arg[11..-1]
+    elsif name = ARGV[1]?
+      lib_name = name
+    else
+      abort "FATAL: library name argument was specified but no value was received.\n#{Help}", USAGE_EXIT_CODE
+    end
+  when .starts_with? "--parent-module"
+    abort "FATAL: only one module name can be specified", USAGE_EXIT_CODE unless mod_name.nil?
```

Comment:
The `.nil?` can be dropped. Conditions check if the value isn't nil or not false.